mlir/include/mlir/Analysis/Presburger/IntegerRelation.h

```diff
@@ -511,6 +511,31 @@ class IntegerRelation {
   void projectOut(unsigned pos, unsigned num);
   inline void projectOut(unsigned pos) { return projectOut(pos, 1); }
 
+  /// The set of constraints (equations/inequalities) can be modeled as an
+  /// undirected graph where:
```

Comment:
Can you write a 1-2 line summary at the top, so that people can quickly understand what the function does, while keeping the details below? You can write that the function removes some constraints that do not impose any bound on the specified variable.